.travis.yml

```diff
@@ -41,9 +41,6 @@ branches:
   except:
   - /^v\d/
 
-before_install:
-  - wget https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh
```

Comment:
Do we maybe need this for future jdk versions?

Reply:
We can always bring it back.

Reply:
Travis under the hood uses that script (in some version).

Reply:
We can add openjdk-ea in order to use java12 early access builds.

Reply:
Good idea with Java 12 EA